### ISSUE TITLE
7.13.6. Screen Name/Title: API documentation - Client libraries - Code Tabs Not Keyboard Accessible

### ISSUE THREAD

**Issue by jzbahrai** [cds-snc/notification-planning#2306]
Components: 
For GC Notify: https://notification.canada.ca/
For GOV.UK: https://www.notifications.service.gov.uk/
For all client links (Java, .NET, NodeJS, etc.)
Observation: 
1. Outbound link icons lack accessible labeling
External link icons (the arrow-out icon in the <svg>) are purely decorative but are not hidden from assistive technologies.
2. Code Tabs Not Keyboard Accessible
The tabbed interface (e.g., Java, .NET, PHP, etc.) is built using <button> elements inside <li>, but they are not implemented as a proper WAI-ARIA Tab Interface.
Keyboard users cannot navigate tabs using arrow keys or get feedback on which tab is selected, which makes interaction confusing.

(Attachment:
7.13.6. Outbound link icons lack accessible labeling.png
7.13.6. Code Tabs Not Keyboard Accessible.mp4)

Recommendation: 
1. Mark the <svg> icons as decorative using aria-hidden="true" and focusable="false".
Ensure the (opens new window) text is the only screen reader-accessible label (you’ve correctly used .sr-only already).
Example:
<svg aria-hidden="true" focusable="false" ...>...</svg>
2. Use ARIA roles and properties like:
role="tablist" on the tab container
role="tab" on each tab button
aria-selected="true" on the active tab
role="tabpanel" on the corresponding content area
Add keyboard support for ArrowLeft/ArrowRight.
Example (structure only):
<div role="tablist" aria-label="Client library languages">
  <button role="tab" aria-selected="true" aria-controls="panel-java" id="tab-java">Java</button>
  <button role="tab" aria-selected="false" aria-controls="panel-dotnet" id="tab-dotnet">.NET</button>
</div>
<div role="tabpanel" id="panel-java" aria-labelledby="tab-java">
  
</div>

**Comment by amazingphilippe** [comment]
Maybe a VuePress upgrade fixes this, otherwise it'll need to go in the a11y statement.

**Comment by andrewleith** [comment]
This is partially done.  I think there was some overlap with other cards.

1. ✅ Outbound link icons lack accessible labeling  
   - Outbound links have the text "(opens in a new tab)" as well as the icon 
2. :x: Code Tabs Not Keyboard Accessible
   - This is a bit misleading.  The code blocks ARE keyboard accessible, you just can't use the arrow keys, which in my opinion is a nice to have.  We could leave this card around and improve this one later if you want.

**Comment by amazingphilippe** [comment]
<img width="343" height="69" alt="Image" src="https://github.com/user-attachments/assets/4621f304-f848-4435-b87f-390577c98f8e" />

**Comment by YedidaZalik** [comment]
Discussed with Phil Add to statement this part

Code Tabs Not Keyboard Accessible
This is a bit misleading. The code blocks ARE keyboard accessible, you just can't use the arrow keys, which in my opinion is a nice to have. We could leave this card around and improve this one later if you want.

They're accessible as a tab stop but do not behave like tabs because in this type of interface you would expect to be able to use left and right arrow keys as well, but they do not work in this type of interface, you need to tab through

We should also improve contrast because blue border is not visible enough
